### DISCUSSION TITLE
Beatmap browse colours

### DISCUSSION THREAD

**Discussion by Tenexxt** [ppy/osu#28975] [Ideas]
Some colours of the UI just don't match. Like opening the beatmap browse, you can see that it has a cyan-ish thing when opening it, but in the main menu its a green? and when opening the rankings in the game, it has the same shade as the beatmap browse button has??? 
More visible here: https://github.com/user-attachments/assets/74c1890c-2503-4f49-a4fd-4d0839764ad8

Also, when opening the profile it would be cool if for the same animations the players profile colours would be aplied (this would probably be a hell to make, but this is an idea, not something that automatically has to be in-game)

Not putting this on issues, because its such a small detail that it doesn't have to take space there.

**Comment by Stoppedpuma** [comment]
> Like opening the beatmap browse, you can see that it has a cyan-ish thing when opening it, but in the main menu its a green?

Going to add that the reason it's blue in this case is because the overlays match osu!web colours. While the main menu buttons may not match, I don't think it's too much of an issue? I also prefer the current green over making the button blue, feels more colourful I guess?

**Reply by Tenexxt** [reply]
Thats why i listed it as an idea, and not a issue, because its such a small detail it doesn't matter. I wanted to list this because its just a small thing i noticed.
